plugins/synapticsmst/synapticsmst-device.c

```diff
@@ -419,11 +448,19 @@ synapticsmst_device_enumerate_device (SynapticsMSTDevice *device,
 		break;
 	/* EVB development board */
 	case 0:
+		system = g_strdup (system_type);
+		/* set up GUID
+		 * GUID is MST-$SYSTEMID-$BOARDID
+		 * $BOARDID includes CUSTOMERID in first byte, BOARD in second byte */
+		if (system != NULL)
+			priv->guid = g_strdup_printf ("MST-%s-%u", system, priv->board_id);
```

Comment:
For EVB I think it will be easier to debug in the future if you make this `MST-evb-%u` since `system_type` will be tied to DMI SMBIOS strings on a particular machine.  That means if you move the EVB from one box to another it will require a new GUID for a payload to work.  Whereas if you hardcode what was going into the string from system_type it to `evb` it will stay consistent and always be able to work.